cloudflare/resource_cloudflare_teams_accounts_test.go

```diff
@@ -40,7 +40,17 @@ func TestAccCloudflareTeamsAccountConfigurationBasic(t *testing.T) {
 					resource.TestCheckResourceAttr(name, "block_page.0.header_text", "hello"),
 					resource.TestCheckResourceAttr(name, "block_page.0.background_color", "#000000"),
 					resource.TestCheckResourceAttr(name, "block_page.0.logo_path", "https://example.com"),
+					resource.TestCheckResourceAttr(name, "logging.0.redact_pii", "true"),
+					resource.TestCheckResourceAttr(name, "logging.0.settings_by_rule_type.0.dns.0.log_all", "false"),
+					resource.TestCheckResourceAttr(name, "logging.0.settings_by_rule_type.0.dns.0.log_blocks", "true"),
+					resource.TestCheckResourceAttr(name, "logging.0.settings_by_rule_type.0.http.0.log_all", "true"),
+					resource.TestCheckResourceAttr(name, "logging.0.settings_by_rule_type.0.http.0.log_blocks", "true"),
+					resource.TestCheckResourceAttr(name, "logging.0.settings_by_rule_type.0.l4.0.log_all", "false"),
+					resource.TestCheckResourceAttr(name, "logging.0.settings_by_rule_type.0.l4.0.log_blocks", "true"),
+					resource.TestCheckResourceAttr(name, "proxy.0.tcp", "true"),
+					resource.TestCheckResourceAttr(name, "proxy.0.udp", "false"),
 				),
+				ExpectNonEmptyPlan: true,
```

Comment:
@rexscaria what is the expectation here? having a non-empty plan implies a permadiff or something isn't matching up.

Reply:
if it is this output, this is a current bug that needs to be fixed from another PR. we can merge without this and just fix it in another PR.

```
TF_ACC=1 go test $(go list ./...) -v -run "^TestAccCloudflareTeamsAccount" -count 1 -parallel 1 -timeout 120m -parallel 1
?   	github.com/cloudflare/terraform-provider-cloudflare	[no test files]
=== RUN   TestAccCloudflareTeamsAccountConfigurationBasic
    resource_cloudflare_teams_accounts_test.go:25: Step 1/1 error: After applying this test step, the plan was not empty.
        stdout:


        Terraform used the selected providers to generate the following execution
        plan. Resource actions are indicated with the following symbols:
          ~ update in-place

        Terraform will perform the following actions:

          # cloudflare_teams_account.qgmqrtaguz will be updated in-place
          ~ resource "cloudflare_teams_account" "qgmqrtaguz" {
              - activity_log_enabled = true -> null
                id                   = "f037e56e89293a057740de681ac9abbe"
                # (2 unchanged attributes hidden)





                # (5 unchanged blocks hidden)
            }

        Plan: 0 to add, 1 to change, 0 to destroy.
--- FAIL: TestAccCloudflareTeamsAccountConfigurationBasic (9.03s)
FAIL
FAIL	github.com/cloudflare/terraform-provider-cloudflare/cloudflare	9.578s
?   	github.com/cloudflare/terraform-provider-cloudflare/tools/cmd/changelog-check	[no test files]
?   	github.com/cloudflare/terraform-provider-cloudflare/version	[no test files]
FAIL
make: *** [testacc] Error 1
```

Reply:
fixed